Makefile

```diff
@@ -48,7 +48,7 @@ lint:
 
 .PHONY: install-tools
 install-tools:
-	go get -u golang.org/x/lint/golint
```

Comment:
The -u flag instructs get to use the network to update the named packages and their dependencies. By default, get uses the network to check out missing packages but does not use it to look for updates to existing packages.

I don't think removing this is necessary good. Maybe we should file an issue to restore this after lint fixes?

Reply:
SGTM

Reply:
Yes, the idea is that this is a temporary workaround.

Reply:
#439